core/block.js

```diff
@@ -980,16 +983,20 @@ Blockly.Block.prototype.setPreviousStatement = function(newBoolean, opt_check) {
       opt_check = null;
     }
     if (!this.previousConnection) {
-      goog.asserts.assert(!this.outputConnection,
-          'Remove output connection prior to adding previous connection.');
+      if (this.outputConnection) {
```

Comment:
Should this be `if (!!this.outputConnection)`? I'm not sure if there would be a case that meets one but not the other...

Reply:
(same question in the cases below as well)

Reply:
If statements have an implicit Boolean() in the argument.